### ISSUE TITLE
aws_backup_vault forces new resource when updating tags

### ISSUE THREAD

**Issue by jverhoeks** [terraform-providers/terraform-provider-aws#7801]
### Terraform Version

Terraform 0.11.11
AWS 1.60

### Affected Resource(s)

aws_backup_vault

### Terraform Configuration Files

```hcl
variable "tags { 
    default = {
      environment = "test
   }
}

resource "aws_backup_vault" "backup_vault" {
  name        = "${var.name}"
  kms_key_arn = "${local.kms_arn}"
  tags        = "${merge(var.tags,map("name",var.name))}"
}

```

### Expected Behavior

Altering the tags for the resource like other resources

### Actual Behavior

(forces new resource) and tries to delete the resource which is blocked by existing backup resource
### Steps to Reproduce

1. Create aws_backup_vault resource 
2. `terraform apply`
3. Add Tag 
4. `terraform apply`

### References

* #7166

**Comment by ewbankkit** [comment]
The `aws_backup_vault.tags` attribute is marked as [`ForceNew: true`](https://github.com/terraform-providers/terraform-provider-aws/blob/ed8c373e45b01af45c6f0cda9d984b08891539d2/aws/resource_aws_backup_vault.go#L30).
In fact the [`TagResource`](https://docs.aws.amazon.com/aws-backup/latest/devguide/API_TagResource.html) and [`UntagResource`](https://docs.aws.amazon.com/aws-backup/latest/devguide/API_UntagResource.html) APIs can be used to modify the vault's tags after creation.

**Comment by slapula** [comment]
Oops!  I have some free time now so I'll take a look at this.

**Comment by kterada0509** [comment]
@slapula sorry. I missed your comment and created a pr.

**Comment by slapula** [comment]
@kterada0509 no worries! I was about to submit pretty much the same PR.  You got there first so lets roll with that.

**Comment by bflad** [comment]
Support for `tags` updates has been merged and will release with version 2.2.0 of the Terraform AWS Provider, likely later today.

**Comment by bflad** [comment]
This has been released in [version 2.2.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#220-march-15-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
